tests/test_fluent_session.py

```diff
@@ -234,15 +234,15 @@ def test_fluent_exit(monkeypatch: pytest.MonkeyPatch):
     monkeypatch.delenv("PYFLUENT_LOGGING")
     monkeypatch.delenv("PYFLUENT_WATCHDOG_DEBUG")
     inside_container = os.getenv("PYFLUENT_LAUNCH_CONTAINER")
-    script = (
-        "import ansys.fluent.core as pyfluent;"
-        "solver = pyfluent.launch_fluent(start_watchdog=False);"
-        f'{"print(solver.connection_properties.cortex_host);" if inside_container else "print(solver.connection_properties.cortex_pid);"}'
-        "exit()"
+    import ansys.fluent.core as pyfluent
+
+    solver = pyfluent.launch_fluent(start_watchdog=False)
+    cortex = (
+        solver.connection_properties.cortex_host
+        if inside_container
+        else solver.connection_properties.cortex_pid
     )
-    output = subprocess.check_output(f'python -c "{script}"', shell=True)
-    cortex = output.decode().strip()
-    cortex = cortex if inside_container else int(cortex)
+    solver.exit()
```

Comment:
@seanpearsonuk @mkundu1 

During investigation we found that due to integration of gRPC FTS the output was changed and it is as follows.

![image](https://github.com/ansys/pyfluent/assets/106588300/bf6d6309-1bf2-484d-bef0-448abff57d20)

This output does not contain container ID or PID which we were processing in next steps. So we have modified this code accordingly and now it is passing in both Linux and Windows.

The hang issue is resolved now.

Thank you.